### PR TITLE
Add incident CSV download to event page

### PR DESCRIPTION
**Pull request by nathanathan**
Currently the document properties all have placeholder values. After the new articleId property is merged I will rebase this and use the property to add the real values.

### CODE REVIEW

**Comment by jgoley** [comment]
Oh, I know this is due to the plugin we use to create CSVs needing a UI table, but it is a little weird that we name the button "Download CSV" then they see a modal and are required to click another button to actually download the CSV file. I suppose we could hide the table and show a modal with some sort of indicator that it has been downloaded? But I can see the benefit of showing the table so if we want to keep it, maybe we automatically download the CSV after the modal appears?

**Comment by nathanathan** [comment]
@jgoley I want the modal dialog to appear before the download starts so we can add controls to it for customizing the exported CSV such as hiding/renaming columns and aggregating counts.

**Comment by jgoley** [comment]
Oh that's a good idea. Maybe we should change the text of the button to "Export" to reduce potential confiusion?

File: client/views/incidentReports/incidentReports.jade [hunk 1]
```diff
@@ -6,7 +6,7 @@ template(name="incidentReports")
           +addIncidentReport
         button.open-download-csv.right-action.btn.btn-primary(type="button")
           i.fa.fa-download
-          span Download CSV
+          span Export CSV
```
Comment (jgoley) [inline client/views/incidentReports/incidentReports.jade]: I think we can also remove "CSV"

**Comment by jgoley** [comment]
With the exception of the wording change, the updates look good 👍

**Pull request merged**